vulnerabilities/importers/almalinux.py

```diff
@@ -0,0 +1,237 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# VulnerableCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/vulnerablecode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import json
+import logging
+from pathlib import Path
+from typing import Any
+from typing import Iterable
+from typing import List
+from typing import Optional
+
+from packageurl import PackageURL
+from univers.version_range import RANGE_CLASS_BY_SCHEMES
+from univers.version_range import RpmVersionRange
+from univers.versions import InvalidVersion
+from univers.versions import RpmVersion
+from univers.versions import Version
+
+from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import AffectedPackage
+from vulnerabilities.importer import Importer
+from vulnerabilities.importers.osv import extract_fixed_versions
+from vulnerabilities.importers.osv import get_affected_version_range
+from vulnerabilities.importers.osv import get_published_date
+from vulnerabilities.importers.osv import get_references
+from vulnerabilities.importers.osv import get_severities
+from vulnerabilities.utils import build_description
+from vulnerabilities.utils import dedupe
+from vulnerabilities.utils import get_advisory_url
+from vulnerabilities.utils import get_cwe_id
+
+logger = logging.getLogger(__name__)
+BASE_URL = "https://github.com/AlmaLinux/osv-database"
+
+
+class AlmaImporter(Importer):
+    spdx_license_expression = "MIT License"
+    license_url = "https://github.com/AlmaLinux/osv-database/blob/master/LICENSE"
+    importer_name = "Alma Linux Importer"
+
+    def advisory_data(self) -> Iterable[AdvisoryData]:
+        try:
+            self.clone(repo_url=self.BASE_URL)
+            base_path = Path(self.vcs_response.dest_dir)
+            advisory_dirs = base_path / "tree/master/advisories"
```

Comment:
@ambuj-1211 This path isn't functioning; the importer isn't actually importing any data.

```suggestion
            advisory_dirs = base_path / "advisories/"
```

Please ensure you run the importer and verify the data is correctly imported.